src/compas_view2/app/controller.py

```diff
@@ -123,13 +123,13 @@ def mouse_key_check(self, event, key_status: Dict, mouse_key: Dict):
             The dictionary that is like `"mouse_key": { "mouse": "right", "key": "shift" }`.
 
         Returns
-        ----------
+        -------
```

Comment:
i don't think there should a `:` after `bool`

Reply:
thanks. changed :)